bundles/org.openhab.binding.amazonechocontrol/src/main/resources/OH-INF/i18n/amazonechocontrol.properties

```diff
@@ -163,10 +163,18 @@ channel-type.amazonechocontrol.subtitle1.label = Subtitle 1
 channel-type.amazonechocontrol.subtitle1.description = Subtitle 1
 channel-type.amazonechocontrol.subtitle2.label = Subtitle 2
 channel-type.amazonechocontrol.subtitle2.description = Subtitle 2
+channel-type.amazonechocontrol.relativeHumidty.label = Humidity
```

Comment:
This doesn't look alphabetized, can you run the i18n tool, so that the diff next time won't include your changes being moved? For reference: https://next.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file

Reply:
I've generated the default translations.